l10n_it_vat_statement_communication/models/account.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class AccountTaxCode(models.Model):
+    _inherit = "account.tax"
+
+    vsc_exclude_operation = fields.Boolean(
+        string='Exclude from active / passive operations')
+
+    vsc_exclude_vat = fields.Boolean(
+        string='Exclude from VAT payable / deducted')
```

Comment:
Guardando meglio questi campi, ho l'impressione che si vadano poi eventualmente a sovrapporre con [l10n_it_vat_statement_communication_reverse_charge](https://github.com/OCA/l10n-italy/tree/af4cbe73ea54b12f4c7e272ae503d2786017bc9d/l10n_it_vat_statement_communication_reverse_charge)

@alessandrocamilli @labaggio possibile?
Qual è il caso d'uso di questi campi? Quando potrebbero essere valorizzati? Con che combinazioni?
Se li teniamo, sarebbero comunque da documentare.

Forse sarebbe meglio rilasciare la fix per `CodiceFornitura` e poi valutare se e come aggiungere questi campi

Reply:
Ecco la PR https://github.com/OCA/l10n-italy/pull/1294

Reply:
ok, quindi proccedo io modifico questa, togliendo la parte relativa al codiceFornitura

Reply:
Da decidere se chiudere questa in favore della https://github.com/OCA/l10n-italy/pull/1249